openpype/hosts/maya/plugins/publish/collect_pointcache.py

```diff
@@ -43,17 +43,29 @@ def process(self, instance):
             instance.remove(proxy_set)
             instance.data["setMembers"].remove(proxy_set)
 
-        # Collect user defined attributes.
-        if not instance.data.get("includeUserDefinedAttributes", False):
-            return
+        # Backwards compatibility for attributes.
+        backwards_mapping = {
+            "write_color_sets": "writeColorSets",
+            "write_face_sets": "writeFaceSets",
+            "include_user_defined_attributes": "includeUserDefinedAttributes"
+        }
+        for key, value in backwards_mapping.items():
+            if key in instance.data:
+                instance.data[value] = instance.data[key]
```

Comment:
Should we maybe change it to this:
```suggestion
        for key, value in backwards_mapping.items():
            if key in instance.data:
                self.log.debug(
                    "Using legacy attribute name '{}' since it exists.".format(key)
                )
                instance.data[value] = instance.data.pop(key)
```

So that at least the log mentions the usage of the legacy attribute + that we pop the old attribute from the instance data here so we know for a fact other plugins are not using the old key by accident?